<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortalTest</class>
 <widget class="QMainWindow" name="PortalTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDE Portal test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Screen sharing:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="screenShareCombobox">
        <item>
         <property name="text">
          <string>Monitor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Window</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="screenShareButton">
        <property name="text">
         <string>Request</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
